invenio_records_rest/views.py

```diff
@@ -403,13 +403,22 @@ def get(self, **kwargs):
                 size * page < self.max_result_window:
             links['next'] = url_for(endpoint, page=page + 1, **urlkwargs)
 
-        return self.make_response(
+        response = self.make_response(
             pid_fetcher=self.pid_fetcher,
             search_result=search_result.to_dict(),
             links=links,
             item_links_factory=self.item_links_factory,
         )
 
+        # Add Link HTTP header
```

Comment:
This fits better in [record_responsify](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/serializers/response.py#L48-L49) and [search_responsify](https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/serializers/response.py#L68-L69).